packages/flutter_tools/test/general.shard/fake_process_manager_test.dart

```diff
@@ -3,6 +3,8 @@
 // found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:convert' show utf8;
+import 'dart:io';
```

Comment:
Can you instead import https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/base/io.dart ?

Reply:
Done. Also done for dart:convert.